packages/dashboard/src/customization/widgets/triangleSymbol/plugin.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import TriangleWidgetComponent from './component';
+import TriangleIcon from './icon';
+import type { DashboardPlugin } from '~/customization/api';
+import { TriangleWidget } from '../types';
+
+export const trianglePlugin: DashboardPlugin = {
+  install: ({ registerWidget }) => {
+    registerWidget<TriangleWidget>('triangle', {
```

Comment:
To go along with my comment about the organization, a plugin doesn't only need to be 1 component. We could have a symbol library plugin which registers all of the symbol widgets instead of just 1.

Reply:
i don't think we can have a single plugin for all widgets, since the `WidgetRegistrationOptions` argument to `registerWidget` allows us to register only a single widget in the component palette. we could, however, put all the plugins in a single file and return them in an array, and we can use `forEach` in api.ts to register them. does that sound good?

Reply:
What if you called register widget multiple times? We could have something like symbol plugin that calls register widget 4+ times for: line, rectangle, ellipse, triangle, etc.

Reply:
right, that's a better idea. i will try to do that